test/test.html

```diff
@@ -19,7 +19,7 @@
   import test from "test";
   console.log(test);
 </script>
-<script type="module" src="../src/es-module-shims.js"></script>
+<script type="module" src="../dist/es-module-shims.js"></script>
```

Comment:
Was this necessary for the tests to pass? I must admit I quite like being able to run the tests without a rebuild.

Reply:
Short answer: yes, this is necessary for tests to pass.
Long answer: The `importScript()` method used in WorkerShim constructor fails if the script it loads has ES6 syntax; as far as I know the `importScripts()` method will automatically fail inside module workers.
```
const workerScriptUrl = createBlob(\`importScripts('${es_module_shims_src}'); 
    self.importMap = ${JSON.stringify(options.importMap || {})}; importShim('${new URL(aURL, pageBaseUrl).href}')`);
```

Reply:
Ah I see what you mean, thanks.

Let's go with (2) then so that this test restriction only applies to the worker tests.